hotsos/core/plugins/rabbitmq/report.py

```diff
@@ -73,11 +73,10 @@ def skewed_nodes(self):
             # Report the node with the greatest skew of queues/vhost
             if skewed_queue_nodes:
                 max_node = None
-                for node_name in skewed_queue_nodes:
+                for node_name, count in skewed_queue_nodes.items():
```

Comment:
Not related to the change but could simplify this.

This block:
```
                max_node = None
                for node_name, count in skewed_queue_nodes.items():
                    if max_node is None:
                        max_node = node_name
                    elif count >= skewed_queue_nodes[max_node]:
                        max_node = node_name
```

can be replaced with:
```
    max_node = max(skewed_queue_nodes, key=skewed_queue_nodes.get)
```

Reply:
Here you go: https://github.com/canonical/hotsos/pull/824

Apologies in advance, @brianphaley, for causing a conflict ;-)